ui/css/perf.css

```diff
@@ -664,3 +664,24 @@ li.pagination-active.active > button {
 .tests-table .rt-tr-group {
   flex: initial;
 }
+
+.alert-title-container {
+  width: 250px;
+}
+
+.alert-title-container .alert-title {
+  overflow-wrap: break-word !important;
+}
+
+.tags-and-options-container {
+  width: fit-content;
+}
+
+.tags-and-options-container span {
+  white-space: pre-wrap;
+  text-align: left;
+}
+
+.tags-and-options-td {
+  padding: 0 !important;
```

Comment:
[question] and here

Reply:
We can use boostrap class `p-0`

Reply:
we sticked with a custom class because we needed a padding top added.